src/Core/Entities/Collection.cs

```diff
@@ -1,15 +1,17 @@
 ﻿using System.ComponentModel.DataAnnotations;
 using Bit.Core.Utilities;
 
+#nullable enable
+
 namespace Bit.Core.Entities;
 
 public class Collection : ITableObject<Guid>
 {
     public Guid Id { get; set; }
     public Guid OrganizationId { get; set; }
-    public string Name { get; set; }
+    public string Name { get; set; } = null!;
```

Comment:
question: Should we really be doing this when enabling nullabe? Seems like the whole point of having strict null checks?

Reply:
It's a _good_ solution, we have to promise to the compiler that when this is created through reflection, like how dapper and EF Core do it, that it will be given a non-null value. 

The better solution is to have the `required` keyword put onto the property but unfortunately that causes some errors that I would need to resolve (mostly in tests) but with trying to keep this series of PR's small and only targeting a single team each it wasn't possible right now. I definitely encourage teams to use that going forward.